Tensile/KernelWriterAssembly.py

```diff
@@ -1387,6 +1389,17 @@ def initKernel(self, kernel, tPA, tPB ):
     vgprIdx = 0
 
     self.startVgprValuC = vgprIdx; vgprIdx += self.numVgprValuC
+    if kernel["MatrixInstruction"]:
+      # MI kernels can overlap C-tile w/ AB-tile up until writeback. Illustrated below:
+      # |<-------------- valuC -------------->|
+      # |------------|-----------|xx|---------|
+      #   lastValuAB ^           ^  ^         ^
+      #         lastVgprForReads ^  ^         ^
+      #              startVgprReuse ^         ^
+      #                             lastValuC ^
+      # TODO a bit tricky. Better to manage all GPRs solely through RegisterPool
+      self.overlapVgprC = True
+      vgprIdx = 0
```

Comment:
vgprIdx already assigned 0 above, this line isn't needed unless you're adding it for emphasis or clarity.

Reply:
It's actually needed because it is incremented in line 1391. But I see your point here: it wasn't obvious at first sight. I will make it more explicit here

Reply:
Oh thanks, I didn't see that there. It's not part of your changeset, but that should really get cleaned up at some point: no really good reason to have multiple lines of code on a single line.